examples/Multiplex/Sample/ScreenNode.m

```diff
@@ -0,0 +1,156 @@
+//
+//  ScreenNode.m
+//  Sample
+//
+//  Created by Huy Nguyen on 16/09/15.
+//  Copyright (c) 2015 Facebook. All rights reserved.
+//
+
+#import "ScreenNode.h"
+#import "AsyncDisplayKit/AsyncDisplayKit.h"
+
+@interface ScreenNode() <ASMultiplexImageNodeDataSource, ASMultiplexImageNodeDelegate, ASImageDownloaderProtocol>
+@end
+
+@implementation ScreenNode
+
+- (instancetype)init
+{
+  if (!(self = [super init])) {
+    return nil;
+  }
+
+  // multiplex image node!
+  // NB:  we're using a custom downloader with an artificial delay for this demo, but ASBasicImageDownloader works too!
+  _imageNode = [[ASMultiplexImageNode alloc] initWithCache:nil downloader:self];
+  _imageNode.dataSource = self;
+  _imageNode.delegate = self;
+  
+  // placeholder colour
+  _imageNode.backgroundColor = ASDisplayNodeDefaultPlaceholderColor();
+  
+  // load low-quality images before high-quality images
+  _imageNode.downloadsIntermediateImages = YES;
+  
+  // simple status label
+  _textNode = [[ASTextNode alloc] init];
+  
+  [self addSubnode:_imageNode];
+  [self addSubnode:_textNode];
+  
+  return self;
+}
+
+- (void)start
+{
+  [self setText:@"loading…"];
+  _textNode.userInteractionEnabled = NO;
+  _imageNode.imageIdentifiers = @[ @"best", @"medium", @"worst" ]; // go!
+}
+
+- (void)reload {
+  [self start];
+  [_imageNode reloadImageIdentifierSources];
+}
+
+- (void)setText:(NSString *)text
+{
+  NSDictionary *attributes = @{NSFontAttributeName: [UIFont fontWithName:@"HelveticaNeue-Light" size:22.0f]};
+  NSAttributedString *string = [[NSAttributedString alloc] initWithString:text
+                                                               attributes:attributes];
+  _textNode.attributedString = string;
+  [self setNeedsLayout];
+}
+
+- (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
+{
+  return
+  [ASInsetLayoutSpec
```

Comment:
I would put this first line on the same line as the return.  More importantly, indent the following lines by 2 spaces per level, rather than just one.